### DISCUSSION TITLE
Wiznet release

### DISCUSSION THREAD

**Discussion by PDE-Paul** [espressif/arduino-esp32#9592] [Q&A]
Hi, I wish to know when will the following examples be implemented as I am waiting for the Wiznet release, I just updated to 2.0.16 and the Wiznet example is not installed again

![image](https://github.com/espressif/arduino-esp32/assets/62668857/afd52f90-103e-4d92-8e9a-1e123b232a99)

 but it shows in GitHub or am I missing something?

![image](https://github.com/espressif/arduino-esp32/assets/62668857/d203b750-912d-46c9-ac9e-21ce9344e06a)

Regards
Paul

**Comment by lucasssvaz** [comment]
Master branch of the repository tracks the most recent changes. This example can be accessed now by using the 3.0.0 pre-release version.

**Comment by PDE-Paul** [comment]
Thank you
